tests/mypy_test.py

```diff
@@ -531,19 +548,24 @@ def main() -> None:
     exclude = args.exclude or []
     code = 0
     total_files_checked = 0
+    total_packages_skipped = 0
     with tempfile.TemporaryDirectory() as td:
         td_path = Path(td)
         for version, platform in product(versions, platforms):
             config = TestConfig(args.verbose, filter, exclude, version, platform)
-            code, files_checked_this_version = test_typeshed(code, args=config, tempdir=td_path)
+            code, files_checked_this_version, packages_skipped_this_version = test_typeshed(code, args=config, tempdir=td_path)
             total_files_checked += files_checked_this_version
+            total_packages_skipped += packages_skipped_this_version
     if code:
         print_error(f"--- exit status {code}, {total_files_checked} files checked ---")
         sys.exit(code)
-    if not total_files_checked:
+    if total_packages_skipped:
+        print(colored(f"--- {total_packages_skipped} packages skipped ---", "yellow"))
```

Comment:
Grammar nit:

```suggestion
        plural = "" if total_packages_skipped == 1 else "s"
        print(colored(f"--- {total_packages_skipped} package{plural} skipped ---", "yellow"))
```

Could possibly do similar for the `total_files_checked` print() immediately below, as well :)

Reply:
and above :)